src/LaunchDarkly.Logging.Microsoft/LaunchDarkly.Logging.Microsoft.csproj

```diff
@@ -29,7 +29,7 @@
 
   <ItemGroup>
     <PackageReference Include="LaunchDarkly.Logging" Version="[1.0.1,]" />
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="[6.0.0,7.0.0)" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="[6.0.0,]" />
```

Comment:
I think that this is the only thing we would need to change to be usable for with versions of Microsoft.Extensions.Logging, but I am not comfortable with open ended versions. I would be comfortable with 8.0 as it will be releasing soon. Assuming that it works with the preview version of Microsoft.Extensions.Logging.

If someone wants to use it with 9, for instance, before we verify 9, then a binding redirect can be used moving the responsibility of version compatibility to the user.

Or if microsoft is providing documented guidance that they won't break compatibility.

Reply:
Resolved per PR